.github/actions/setup-docker-ssl/action.yml

```
@@ -0,0 +1,55 @@
+name: 'Setup Docker with SSL'
+description: 'Setup Docker Buildx, install mkcert, and generate SSL certificates'
+
+inputs:
+  certificate-domains:
+    description: 'Domains for SSL certificate (space-separated)'
+    required: false
+    default: 'localhost 127.0.0.1 ::1'
+  ssl-directory:
+    description: 'Directory to store SSL certificates'
+    required: false
+    default: 'ssl'
+  cert-filename:
+    description: 'Certificate filename (without extension)'
+    required: false
+    default: 'music-manager'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      
+    - name: Install mkcert
```

optional but more informative
```suggestion
    - name: Install mkcert (if not present)
```